### ISSUE TITLE
Build

### ISSUE THREAD

**Issue by jiauka** [KimBP/VHFtoN2k-blue#1]
Hi:

I'm trying to build this device.

Which IDE did you use to build it. I know it's eclipse based, but I can compile it with AC6 or Atollic.

thanks!
j.

**Comment by KimBP** [comment]
I've used a plain Eclipse Neon (running at Ubuntu) with plugins like OpenOCD debugging and STM32Fx Project templates.

I'm not sure if you are missing a 'not' (a.k.a. can not compile..) in your issue. I guess you do, otherwise it wouldn't make sense to create the issue. 
I don't know neither AC6 or Atollic, but what problems do you experience?

**Comment by jiauka** [comment]
thanks a lot, yes, a not was missing.

I will try plain Eclipse Neon

**Comment by jiauka** [comment]
I manage  to build it, but it fails with a hard fault.  Probably related to eclipse/nano-libc/arm-gcc setup.

Anyway, I'm working on a NON-MCP2515 solution, so far it works, just need minor fixes, if you are interested, drop me a note or email me to jiauka___AT__gma...dotcom

**Comment by KimBP** [comment]
With NON-MCP2515 solution, do you then mean using the ST32F103 build in CAN?
I once had an idea of using that my self but hadn't had the time (motivation) for doing anything serious. So yes I'm interested.

**Comment by jiauka** [comment]
Yes, using the internal. I have it working, It needs some code cleanup and change the TX stuff to a interrupt based system.

You can download it at
 https://github.com/jiauka/NMEA2000_stm32f1
